captum/optim/_param/image/transforms.py

```diff
@@ -10,8 +10,6 @@
 from captum.optim._utils.image.common import nchannels_to_rgb
 from captum.optim._utils.typing import IntSeqOrIntType, NumSeqOrTensorType
 
```

Comment:
The `CenterCrop`, `RandomScale`, & `RandomSpatialJitter` classes should have the line: `super(ClassName).__init__()` changed to: `super().__init__()`, as I think that's the correct Python 3 format.